test/shell/cli_test.sh

```diff
@@ -18,13 +18,13 @@
 #
 
 start_adapter() {
-  run ./etcd-adapter
+  run systemctl start etcd-adapter
```

Comment:
Where is the command definition of `run`?

Reply:
@tokers Which is defined by the https://github.com/sstephenson/bats.git framework. 

https://github.com/sstephenson/bats/blob/03608115df2071fff4eaaff1605768c275e5f81f/libexec/bats-exec-test#L50